Userland/Libraries/LibCore/Account.cpp

```diff
@@ -125,6 +125,35 @@ ErrorOr<Account> Account::from_uid(uid_t uid, [[maybe_unused]] Read options)
     return from_passwd(*pwd, spwd);
 }
 
+ErrorOr<Vector<Account>> Account::all([[maybe_unused]] Read options)
+{
+    Vector<Account> accounts;
+
+    ScopeGuard pwent_guard([] { endpwent(); });
+    setpwent();
+    errno = 0;
+    while (auto const* pwd = getpwent()) {
+        spwd spwd = {};
+
+#ifndef AK_OS_BSD_GENERIC
```

Comment:
Do all systems except `AK_OS_BSD_GENERIC` really have `getspnam()` and friends?

Reply:
Pretty sure Windows doesn't. I'll add a check for that

Reply:
(And Android doesn't have it in bionic libc either...)

Reply:
Actually @ADKaster pointed out this file is already excluded from Windows and Android - so looks like we can drop that commit as this won't be an issue either way.

https://github.com/SerenityOS/serenity/blob/master/Userland/Libraries/LibCore/CMakeLists.txt#L37-L45